R/gen_functions.R

```diff
@@ -143,6 +143,8 @@ gen_temperature <- function(world,
 
   map_x <- max(world$map$x)
   map_y <- max(world$map$y)
+  seed <- ifelse(is.null(noise) || is.null(noise$seed), 
+                 sample.int((2^31 - 1), 1), noise$seed)
```

Comment:
expanded ifelse() call to be easier to read.

```suggestion
  if (is.null(noise) || is.null(noise$seed)) {
    seed <- sample.int((2^31 - 1), 1)
  } else {
    seed <- noise$seed
  }
```